README.md

```diff
@@ -1,2 +1,142 @@
-# paps
-News Room challenge
+# Pap News 
+
+Welcome to Pap News - _Spread the knowledge!_
+
+## Description
+Week 8 Project of Craft Academy cohort January 2018.
+
+PapsNews is a newsagency Web Application used to share your local and international news directly on your device screen.  
+
+Once you land on the Homepage (https://....herokuapp.com/), you need to Signup in order to have access to the web contents
+(click on sign up in the navbar), you will automatically have 7 days free trial, at the end of which you can become a subscriber for a minimum period of 12 months.
```

Comment:
Are we sure about this trial functionality?

Reply:
We agreed this morning, remember? Anyway, it's a work in progress document, it will be updated everyday.

Reply:
OK, I must have misunderstood something, No probs.